animations/shared-element-transitions/index.md

```diff
@@ -0,0 +1,684 @@
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* A radial transformation_ animates a circular element into a square
+  element.
+* Because this is a Hero animation, the Hero transforms while
+  flying in an overlay from the source route to the destination route.
+* MaterialRectCenterArcTween defines the tween animation.
+* Build the destination route using PageRouteBuilder.
+</div>
+
+Animating an element as it transforms from a circular shape to a
+rectanglar shape is a slick effect that you can implement using Hero
+widgets. To accomplish this, the code animates the intersection of
+two clip shapes: a circle and a square.
+Throughout the animation, the circle clip (and the image) scales from
+`minRadius` to `maxRadius`, while the square clip maintains a constant
+size. At the same time, the image flies
+from its position in the source route to its position in the
+destination route. For visual examples of this transition, see
+<a href="https://material.io/guidelines/motion/transforming-material.html#transforming-material-radial-transformation">Radial
+transformation</a> in the Material motion spec.
+
+This animation might seem complex (and it is), but you can
+**customize the provided example to your needs.**
+The heavy lifting is done for you.
+
+<aside class="alert alert-info" markdown="1">
+**Show me the code!**
+
+Each of the following examples demonstrates a radial transformation.
+This guide describes the first example.<br><br>
+
+[radial_transition](https://github.com/flutter/website/tree/master/_includes/code/animation/radial_transition)
+: A radial transformation as described in the Material motion spec.
+
+[basic_radial_transition](https://github.com/flutter/website/tree/master/_includes/code/animation/basic_radial_transition)
+: A radial transformation with a simple destination route&mdash;it has
+  no Scaffold, Card, Column, or Text.
+  This basic example, provided for your reference, isn't
+  described in this guide.
+
+[radial_transition_animate_rectclip](https://github.com/flutter/website/tree/master/_includes/code/animation/basic_radial_transition_animate_rectclip)
+: Extends radial_transition by also animating the size of the
+  rectangular clip.
+  This more advanced example, provided for your reference, isn't
+  described in this guide.
+
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Pro tip:**
+The radial animation involves intersecting a round shape with
+a square shape. This can be hard to see, even when slowing
+the animation with `timeDilation`, so you might consider enabling
+Flutter's [visual debugging mode](/debugging/#visual-debugging)
+during development.
+</aside>
+
+### What's going on?
+
+The following diagram shows the clipped image at the beginning
+(`t = 0.0`), and the end (`t = 1.0`) of the animation.
+
+<img src="images/radial-transition.png" alt="visual diagram of
```

Comment:
You could achieve total symmetry maximization by reusing the ClipOval label in the center to point to the clip oval on the right :-)

Reply:
LOL. Yes, I fixed it!